test/e2e/data/infrastructure-aws/withclusterclass/e2e_test_templates/cluster-template-external-vpc-clusterclass.yaml

```diff
@@ -0,0 +1,1078 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
This file is not required in templates, as we are not running this test.

Reply:
Will remove. Thanks.